apps/sports-helsinki/src/domain/search/eventSearch/searchResultList/OrderBySelect.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  OrderBy,
+  OrderDir,
+  useSearchTranslation,
+  useUnifiedSearch,
+  Select,
+  SmallSpinner,
+  useGeolocation,
+} from 'events-helsinki-components';
+import type { GeolocationContextType } from 'events-helsinki-components';
+import React from 'react';
+import useHandleOrderChange from '../../../../hooks/useHandleOrderChange';
+import styles from './orderBySelect.module.scss';
+
+const OrderBySelect: React.FC = () => {
+  const { t } = useSearchTranslation();
+  const { filters } = useUnifiedSearch();
+  const geolocation: GeolocationContextType = useGeolocation({ skip: true });
+  const handleOrderChange = useHandleOrderChange();
+
+  const defaultOption = {
+    text: t('search:order_by.alphabetical'),
```

Comment:
trivial, but haven't we used lowCamelCases with rest of the translation keys?

Reply:
Searching with underscore under locales gives some matches not related to this pull request but most translation keys are not using underscore. Taking into account.